src/Blockcore.Explorer/ClientApp/src/app/app.module.ts

```diff
@@ -30,6 +30,8 @@ import { AddressComponent } from './explorer/address/address.component';
 import { InsightComponent } from './insight/insight.component';
 import { RichlistComponent } from './insight/richlist/richlist.component';
 import { TippyDirective } from './shared/tippy.directive';
+import { ContractTransactionComponent } from './explorer/contracttransaction/contracttransaction.component';
```

Comment:
It's common pattern to use "-" when concatenating words on files and paths in Angular.

`contract-transaction`

Reply:
ah ok yes I was not sure I will change